SU-SWS/stanford_basic
# 8.4.2

# READY FOR REVIEW 

# Summary
- Upgrade to Decanter 6.0.4
- Ready for cutting tag.

# Review By (Date)
- Today

# Urgency
- Today

# Steps to Test

1. Review changes to the code and call out anything that doesn't look right.
2. Check out branch locally
3. Compile assets (nvm use, npm install, npm run publish)
4. Validate no change to git repo happens after asset compile. 
5. Clear cache in your site and validate the search icon is red and the button is square.

# Affected Projects or Products
- D8CORE RC5 release

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
